README.md

```diff
@@ -1334,6 +1334,66 @@ Reset the location service on real device since Appium 1.22.0.
 It could delay a few seconds to reflect the location by the system.
 It raises an error if the device is simulator or an error occurred during the reset.
 
+### mobile: enableConditionInducer
+
+Important: Device conditions are available for real devices running iOS 13.0 and later.
+
+You can create a device condition on a connected device to test your app under adverse conditions, such as poor network connectivity or thermal constraints.
+
+When you start a device condition, the operating system on the device behaves as if its environment has changed. The device condition remains active until you stop the device condition or disconnect the device. For example, you can start a device condition, run your app, monitor your app’s energy usage, and then stop the condition.
+
+Reference: [Test under adverse device conditions (iOS)](https://help.apple.com/xcode/mac/current/#/dev308429d42)
+
+Name | Type | Required | Description                                                                                                                                                                                                                                        | Example
+--- | --- | --- |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ---
```

Comment:
extra dashes are not needed. Please check how other tables are formatted in this document